### PR TITLE
Adding payment

### PR DESCRIPTION
**Pull request by maximilian02**
- Added modules needed for cart/payment module
- Started to use [path.sep](https://nodejs.org/api/path.html#path_path_sep) to avoid slashes problem on different OS context.

https://github.com/MakingSense/mean-seed/issues/52

### CODE REVIEW

**Comment by mravinale** [comment]
awesome!! :)

File: cart/templates/payments.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+module.exports = simpleDI.inject(['stripe'], function (stripe) {
+
+    var stripeObject = stripe(process.env.STRIPE_PUBLISHABLE_KEY);
+
+    return {
+
+        stripe: function (req, res) {
+            if(!req.body.id || !req.body.amount || !req.body.currency) {
+                return res.json(404, {
+                    message: 'Field not found. Token id, amoung and currency are required.'
```
Comment (AlphaGit) [inline cart/templates/payments.js]: Typo: should say "amount".

Comment (mravinale) [inline cart/templates/payments.js]: good catch! :) bionic eye

Comment (maximilian02) [inline cart/templates/payments.js]: Nice catch.

**Pull request merged**